<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaItemView</class>
 <widget class="QWidget" name="MediaItemView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>552</width>
    <height>65</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="2" column="0" colspan="2">
    <widget class="QWidget" name="containerWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>510</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <property name="verticalSpacing">
       <number>0</number>
      </property>
      <item row="0" column="0" rowspan="2">
       <widget class="QLabel" name="thumbnailLabel">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../resource/image.qrc">:/image/thumbnail-default</pixmap>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QWidget" name="titleWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="checkedBox">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="titleLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Title</string>
           </property>
           <property name="indent">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="infoWidget" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <property name="spacing">
          <number>5</number>
         </property>
         <property name="leftMargin">
          <number>3</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="durationIconLabel">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../resource/image.qrc">:/image/item-time</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="durationLabel">
           <property name="styleSheet">
            <string notr="true">color: rgb(120, 120, 120);</string>
           </property>
           <property name="text">
            <string>0:00:28</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="urlIconLabel">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../resource/image.qrc">:/image/item-link</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="urlLabel">
           <property name="text">
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;http://www.video.com/&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="openExternalLinks">
            <bool>true</bool>
           </property>
           <property name="textInteractionFlags">
            <set>Qt::LinksAccessibleByMouse</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="parsingIconLabel">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../resource/image.qrc">:/image/item-convert</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="parsingLabel">
           <property name="styleSheet">
            <string notr="true">color: rgb(120, 120, 120);</string>
           </property>
           <property name="text">
            <string>Parsing...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="errorIconLabel">
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="../resource/image.qrc">:/image/item-error</pixmap>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="errorLabel">
           <property name="styleSheet">
            <string notr="true">color: rgb(120, 120, 120);</string>
           </property>
           <property name="text">
            <string>Error</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>1</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/image.qrc"/>
 </resources>
 <connections/>
</ui>
